testpilot/experiments/serializers.py

```diff
@@ -55,6 +56,9 @@ def get_installations_url(self, obj):
                        args=(obj.pk,))
         return request.build_absolute_uri(path)
 
+    def get_survey_url(self, obj):
+        return 'https://qsurvey.mozilla.com/s3/%s' % obj.slug
```

Comment:
Just to make sure I'm understanding the survey process: Can we rely on this always being the URL pattern? If so, that's cool... one less field to add to the DB